README.md

```diff
@@ -8,157 +8,155 @@
-      available inside the examples folder. To use this, you can use the command
-        - `docker run -v ./examples:/edge/data -p 3063:3063 -e BOOTSTRAP_FILE=/edge/data/features.json -e TOKENS='my-secret-123,another-secret-789' unleashorg/unleash-edge:<version> offline`
+To run Edge in **edge** mode, use the tag `edge`. This is built from `HEAD` on each commit.
 
-### Cargo/Rust
-
-If you have the [Rust toolchain](https://rustup.rs) installed you can build a binary for the platform you're running by
-cloning this repo and running `cargo build --release`. This will give you an `unleash-edge` binary in `./target/release`
+```shell
+docker run -p 3063:3063 -e STRICT=true -e UPSTREAM_URL=<your_unleash_instance> -e TOKENS=<your_client_token> unleashorg/unleash-edge:<version> edge
+```
 
-## Concepts
+To run Edge in **offline** mode, use the `offline` tag and provide a volume with your feature toggles file. An example is available inside the examples folder. 
 
-See our page on [Edge concepts](./docs/concepts.md)
+```shell
+docker run -v ./examples:/edge/data -p 3063:3063 -e BOOTSTRAP_FILE=/edge/data/features.json -e TOKENS=<your_client_token_1,your_client_token_2> unleashorg/unleash-edge:<version> offline
+```
 
 ## Metrics
 
-**❗ Note:** For Edge to correctly register SDK usage metrics, your Unleash instance must be v5.9.0 or newer.
-**! Note:** If you're daisy chaining you will need at least Edge 17.0.0 upstream of any Edge 19.0.0 to preserve metrics.
+> Availability: Unleash v5.9+. For daisy-chaining, ensure Edge v17+ is upstream of any Edge v19+ to preserve metrics.
 
-Since Edge is designed to avoid overloading its upstream, Edge gathers and accumulates usage metrics from SDKs for a set
-interval (METRICS_INTERVAL_SECONDS) before posting a batch upstream.
-This reduces load on Unleash instances down to a single call every interval, instead of every single client posting to
-Unleash for updating metrics.
-Unleash instances running on versions older than 4.22 are not able to handle the batch format posted by Edge, which
-means you won't see any metrics from clients connected to an Edge instance until you're able to update to 4.22 or newer.
+Edge is designed to minimize load on its upstream by batching SDK usage metrics. Metrics are gathered over a set interval (`METRICS_INTERVAL_SECONDS`) and sent upstream in a single batch.
 
-## Compatibility
+Unleash versions older than 4.22 cannot process these metrics, so an update is required to see metrics from clients connected to Edge.
 
-Unleash Edge adheres to Semantic Versioning (SemVer) on the API and CLI layers. If you're using Unleash Edge as a
-library in your projects, be cautious, internal codebase changes, which might occur in any version release (including
-minor and patch versions), could potentially break your implementation.
+### Prometheus integration
 
-## Performance
+To monitor the health and performance of your Edge instances, you can consume Prometheus metrics at:
```

Comment:
@sighphyre is this the right way to phrase this?

Reply:
I think so, yes!

Reply:
Can confirm, great job